src/components/NavBar/CircleMenu.css

```diff
@@ -0,0 +1,276 @@
+body {
```

Comment:
Wish I knew css good enough in order to review it...

Reply:
Dont even try